src/frontend/src/selenium.test.ts

```diff
@@ -546,7 +546,7 @@ test("Register new identity and add additional device", async () => {
     await on_Main(DEVICE_NAME1, driver);
     await on_Main(DEVICE_NAME2, driver);
 
-    await await on_Main_Logout(driver);
+    await on_Main_Logout(driver);
```

Comment:
naive question: why is only one await needed here, but in other places (like line 526) two awaits are used?

Reply:
Double await is never *needed* (because of JS's questionable design to auto-flatten Promises) it's basically a No-Op. I'll find the other spots this is happening and correct them as well.